pkgs/by-name/nu/numbat/package.nix

```diff
@@ -1,27 +1,33 @@
 { lib
 , stdenv
+, testers
 , fetchFromGitHub
 , rustPlatform
 , darwin
+, numbat
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "numbat";
-  version = "1.6.3";
+  version = "1.7.0";
 
   src = fetchFromGitHub {
     owner = "sharkdp";
     repo = "numbat";
     rev = "v${version}";
-    hash = "sha256-r6uPe2NL+6r/fKjf0C/5DLdB5YP3SIo8g8EsDxKP/3g=";
+    hash = "sha256-KX/V1cVZoPtkP/ehnvcTeaQavHlHP6NJ7g+FbOaYKC4=";
   };
 
-  cargoHash = "sha256-MPqJjCfIwgK8QigWQYfWAYlg9RNMzF4x+0SprS0raKY=";
+  cargoHash = "sha256-skCvIMz50GqveyawVHicpEqMknvZuYNvp1u+gW4wG8A=";
 
   buildInputs = lib.optionals stdenv.isDarwin [
     darwin.apple_sdk.frameworks.Security
   ];
 
+  passthru.tests.version = testers.testVersion {
```

Comment:
Is this how I works?
As I understand it, that should call "numbat --version" and check if the version number appears anywhere on stdout?

Reply:
That's right, testVersion actually executes `${mainProgram} --version` or something and checks whether the stdout contains the correct version.

https://github.com/NixOS/nixpkgs/blob/38f6bf6c9cad76b8128c914d8818150d4a28e781/pkgs/build-support/testers/default.nix#L56-L78

You can check on local machine with the following command.
```sh
$ nix-build -A numbat.tests

checking outputs of '/nix/store/yf41i4sh2nkhv1j7pmvzlyvbi79klgns-numbat-1.7.0-test-version.drv'...
numbat 1.7.0
/nix/store/r7h5n8ywyxw5jp95wg6jh8xwvzjsg57g-numbat-1.7.0-test-version
```